### DISCUSSION TITLE
docker / proxy mode for accidental IP-Leak but ...

### DISCUSSION THREAD

**Discussion by nerd36** [oxen-io/lokinet#1990] [Q&A] [answered]
I can't use Lokinet all the times because of slow speed. To avoid accidental IP-Leak for political activities I use a browser profile with proxy setting directing traffic to a Lokinet docker container but it is the problem:

https://github.com/mahdi-ln/lokinet-socks/issues/1

**Comment by majestrate** [comment, marked as answer]
by default lokinet will blackhole all lan traffic when the exit mode is on.


 you can disable this via the `[network]` section option `blackhole-routes=false`
